edge-modules/edgehub-proxy/src/main.rs

```diff
@@ -148,17 +144,16 @@ fn run() -> Result<(), Error> {
         let common_name = args
             .value_of("common name")
             .expect("no value for required COMMON_NAME");
-        let expiration = args
-            .value_of("expiration")
-            .expect("no value for required EXPIRATION");
+        let expiration = DateTime::parse_from_rfc3339(
+            args
+                .value_of("expiration")
+                .expect("no value for required EXPIRATION"))?;
```

Comment:
This is likely outside the scope, but it might be nice that if we are going to crash on these expects to have something like "no value for ... please see <insert link for documentation here>"

Reply:
if you're referring to all the expects : https://github.com/Azure/iotedge/pull/5349#discussion_r698701338
Currently making changes to replace them. Will resolve this